src/Apps/Artwork/Components/PrivateArtwork/PrivateArtworkAboutArtist.tsx

```diff
@@ -35,6 +39,10 @@ export const PrivateArtworkAboutArtist: React.FC<PrivateArtworkAboutArtistProps>
 
   // FIXME: Remove (typechecker)
```

Comment:
can we remove this line?

Reply:
done!

Reply:
Ahh @JanaeHijaz - forgot to push as this is still in?